app/styles/mixin.scss

```diff
@@ -9,7 +9,9 @@
 $mixin-left-padding: 22px;
 
 .mixin__name {
-  position: relative;
+  position: -webkit-sticky;
```

Comment:
Rather than trying to get all the prefixes in our styles, should we just use https://github.com/kimroen/ember-cli-autoprefixer?

Reply:
Your call. We use `-webkit` twice in the CSS (including this PR).

Reply:
I suppose it's no big deal then.